Packs/AlibabaActionTrail/Playbooks/Alibaba_ActionTrail_-_multiple_unauthorized_action_attempts_detected_by_a_user.yml

```diff
@@ -0,0 +1,635 @@
+id: Alibaba ActionTrail - multiple unauthorized action attempts detected by a user
+version: -1
+name: Alibaba ActionTrail - multiple unauthorized action attempts detected by a user
+description: |-
+  This playbook investigates an “Alibaba ActionTrail - multiple unauthorized action attempts detected by a user” incident by gathering user and IP information and performing remediation based on the information gathered and received from the user.
```

Comment:
Instead of "Incident" -> should be alert

Reply:
also need to change the playbook readme @TalNos

Reply:
done